interchain-exchange/tutorial/00-intro.md

```diff
@@ -5,7 +5,7 @@ order: 1
 # Introduction
 
 The Interchain Exchange is a module to create buy and sell orders in between blockchains.
-In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains, which enables to swap tokens from one blockchain to another.
+In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.
```

Comment:
`which enables to swap tokens from one blockchain to another` why removing this statement? This was pretty cool

Reply:
Ok, great. I thought it might create other expectations. But if you approve I love to keep it 😁

Reply:
Well it's actually about token swap